js/loader.js

```diff
@@ -54,6 +54,16 @@ var Loader = (function () {
 
 		// Notify core of loaded modules.
 		MM.modulesStarted(moduleObjects);
+
+		// Starting modules also hides any modules that have requested to be initially hidden
+		for (var n in moduleObjects) {
```

Comment:
var is outdated, please use let (or const if its a cosntant)

Reply:
Okay, I think that's now done